<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormProveedorBase</class>
 <widget class="QWidget" name="FormProveedorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GBTitulo">
     <property name="title">
      <string>Proveedor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Razon social:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LERazonSocial"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Direccion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="TEDireccion"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>CUIT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LECUIT"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Telefono fijo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETelFijo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Telefono celular:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETelCel"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
